include/swift/AST/NameLookupTypeIDZone.def

```diff
@@ -15,6 +15,9 @@
 //
 //===----------------------------------------------------------------------===//
 
+// Used to escape commas used within template arguments.
+#define COMMA ,
```

Comment:
Clever, but can we use a using declaration or a separate type to help out here?

Reply:
@CodaFi Hmm, I guess for now we could just declare something `using QualifiedLookupResult = SmallVector<ValueDecl *, 4>`. It might make sense to eventually return `LookupResult` for both qualified and unqualified lookups though, what do you think?